### DISCUSSION TITLE
Improve the readability of the code

### DISCUSSION THREAD

**Discussion by zhixunjie** [etcd-io/etcd#18251] [Ideas]
Whether it is necessary to put such a large section of the annotation into the code, I think it is not necessary, because it greatly affects the readability of the code. I suggest finding a document to collect these annotations uniformly, and then only need to refer to the location of these documents in the code.

**Comment by jmhbnz** [comment]
Hi @zhixunjie - Thanks for raising this idea. Can you please be more specific about which comments/annotation in the code you think are not necessary or should be moved elsewhere?

**Reply by zhixunjie** [reply]
like this large section（about 90+ lines）：
https://github.com/etcd-io/etcd/blob/0223ca52b8d6d4953f708e5e4245c37cb4274115/raft/raft.go#L1127
when i tried to figure out the mechanism about how to deal with the follower's reject here，it made me so confused；
